rbac/internal/views.py

```diff
@@ -695,7 +696,7 @@ def permission_removal(request):
         with transaction.atomic():
             try:
                 logger.warning(f"Deleting permission '{permission}'. Requested by '{request.user.username}'")
-                permission_obj.delete()
+                delete_permission(permission_obj)
```

Comment:
When is the admin endpoint called vs when are permissions removed during seeding?

I'm wondering if the logic for handling replication of permission removal should be done in the seeding handler or at least that module.